devops/docker/mariner-2-amd64/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
+
+RUN tdnf install -y mariner-repos-extended
+RUN tdnf install -y \
+    build-essential \
+    ca-certificates \
+    cmake \
+    curl-devel \
+    file \
+    gcovr \
+    git \
+    gmock-devel \
+    gtest-devel \
+    jq \
+    mariner-rpm-macros \
+    ninja-build \
+    openssl-devel \
+    rapidjson-devel \
+    rpm-build \
+    tar \
+    tmux \
+    tree \
+    unzip \
+    util-linux \
+    wget \
+    zip && \
```

Comment:
Please make sure that we really need all of these to build OSConfig for Mariner. Some seem a bit curious.

Reply:
Yes they are all needed and have also been vetted by MSFT as trusted repos and approved packages

Reply:
For what is needed unzip (line 23)?

Reply:
These overall look more like for building Mariner from source, not for running Mariner in a container. Gave above an example (unzip), there are others.

Reply:
Discussed offline. Thanks Ahmed!